### ISSUE TITLE
Add paths next to references when "Show Call Hierarchy" is run

### ISSUE THREAD

**Issue by luabud** [microsoft/pylance-release#5104]
It'd be nice to add paths next to the function names that show when running "Show Call Hierarchy" to better differentiate where they're coming from. 

The screenshot below is from the [FastAPI repo](https://github.com/tiangolo/fastapi) after installing the dependencies in a .venv,  opening .venv/Lib/site-packages/starlette/responses.py and then selecting "Show call hierarchy" on JSONResponse

![image](https://github.com/microsoft/pylance-release/assets/45497113/c29a2528-9771-4e98-9d9e-4c19dd469d82)

This would match what we have for "show type hierarchy": 
![image](https://github.com/microsoft/pylance-release/assets/45497113/c5e32c58-2d45-442d-997f-3c0cd5f46e92)

**Comment by debonte** [comment]
Dupe of https://github.com/microsoft/pylance-release/discussions/2854? Although for the example I put in there I was thinking that we'd show a class name instead of a path. Maybe we need both? Or one or the other depending on the scenario?

**Comment by luabud** [comment]
Yeah it seems to be a dupe, but good point that the path may not be enough. Maybe both would be a good approach? We seem to support the following format for "Show Type Hierarchy": 

![image](https://github.com/microsoft/pylance-release/assets/45497113/d85a9355-cd01-41a2-bcab-a297410603de)

**Comment by heejaechang** [comment]
This issue has been fixed in prerelease version 2026.3.100, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#20263100-24-August-2026-PreRelease)
